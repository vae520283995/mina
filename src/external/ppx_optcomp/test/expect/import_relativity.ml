let (y : string) = ("goodbye" : string)
