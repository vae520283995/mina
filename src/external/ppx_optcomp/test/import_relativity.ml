[%%import "import_relativity/entry.mlh"]

[%%inject "y", x]

[%%warning "goodbye"]

