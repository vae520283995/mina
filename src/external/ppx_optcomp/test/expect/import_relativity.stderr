File "test/import_relativity/nested/../exit.mlh", line 1, characters 3-10::
Warning Expected warning from exit.mlh
File "test/import_relativity.ml", line 5, characters 3-10::
Warning goodbye
